aws/aws.go

```diff
@@ -1,8 +1,9 @@
 package aws
 
 type AWS struct {
-	KMSKeyArn string `json:"kmsKeyArn" yaml:"kmsKeyArn"`
-	Masters   []Node `json:"masters" yaml:"masters"`
-	Region    string `json:"region" yaml:"region"`
-	Workers   []Node `json:"workers" yaml:"workers"`
+	KMSKeyArn  string   `json:"kmsKeyArn" yaml:"kmsKeyArn"`
+	Masters    []Node   `json:"masters" yaml:"masters"`
+	Region     string   `json:"region" yaml:"region"`
+	Workers    []Node   `json:"workers" yaml:"workers"`
+	PublicKeys []string `json:"publicKeys" yaml:"publicKeys"`
```

Comment:
That does not answer my questions. So again. 

- Where do these keys come from?
- What keys has Kubernetesd to create?